lib/postgrex.ex

```diff
@@ -454,4 +455,18 @@ defmodule Postgrex do
   defp defaults(opts) do
     Keyword.put_new(opts, :timeout, @timeout)
   end
+
+  defp ensure_deps_started!(opts) do
+    ssl_required? = Keyword.get(opts, :ssl, false)
+    if ssl_required? and Enum.any?(:application.which_applications(), fn {app, _desc, _vsn} -> app == :ssl end) do
```

Comment:
`:application.which_applications/0` is a GenServer call which can result in a timeout. I don't think that we need to handle it, but decided to place this comment FYI.

Reply:
The boolean logic is inverted, you want `not any?`. Here is another way to write it though:

    if ssl_required? and List.keymember?(:application.which_applications, :ssl, 0)

Reply:
Sorry for that. I had a bad time trying to run postgrex tests on my development env. Getting socket/psql errors all the time because the database is running within a container. So I'm relying on a Travis in a PR and debug via `iex` calls.

Thanks for `List.keymember?/3`, never used that module 👍.